File: src/model/JobEnum.ts
export enum JobEnum {
    DIRECTOR = 'Diretor(a)',
    FINANCIAL = 'Financeiro',
    SECRETARY = 'Secretária',
    TEACHER = 'Professor(a)'
}

export const JobList: {
    key: string;
    value: string;
}[] = Object.entries(JobEnum).map(([key, value]) => ({ key, value }))

export type JobType = keyof typeof JobEnum

File: src/main/java/br/com/api/enums/JobEnum.java
package br.com.api.enums;

import lombok.AllArgsConstructor;
import lombok.Getter;

@AllArgsConstructor
@Getter
public enum JobEnum {

	TEACHER("label.teacher"), 
	SECRETARY("label.secretary"), 
	FINANCIAL("label.financial"), 
	DIRECTOR("label.director");

	private final String i18n;
}
